week-4/Homework/mandatory/4-groceries.js

```diff
@@ -40,7 +45,10 @@ Exercise 2:
 // Gather weekend item names into this array
 let weekendGroceriesToBuy = [];
 
+let weekendList = weeklyMealPlan.saturday + weeklyMealPlan.sunday;
+weekendGroceriesToBuy.push(weekendList);
 
+console.log(weekendGroceriesToBuy);
```

Comment:
This isn't quite correct. You have the right elements, but as an array of 1 item:
["Olive oil,Potato,Salmon,Asparagus"]
The output should be an array of 4 items:
["Olive oil", "Potato", "Salmon", "Asparagus"]